views/season.pug

```diff
@@ -0,0 +1,21 @@
+html 
+    head 
+        title TV synopsis   
+        link(rel="stylesheet" href="/css/styles.css")
```

Comment:
While this repetition between the two templates doesn't seem like a big deal, as our site gets bigger and more complex maintaining this code in many places can become tedious and error prone. This is where the concept of partials comes in. Partials are parts of a template that have been abstracted out to their own files. These partials can then be included into many pages using Pug's include command. This means the code can live in one place, making it easy to maintain, but be used in many places.